roles/download/defaults/main.yml

```diff
@@ -67,7 +67,7 @@ quay_image_repo: "quay.io"
 
 # TODO(mattymo): Move calico versions to roles/network_plugins/calico/defaults
 # after migration to container download
-calico_version: "v3.18.4"
+calico_version: "v3.19.1"
```

Comment:
As far as I remember, we agree to postpone calico 3.19 until next spray version 🤔

Reply:
This is for master which would become 2.17, so I'm OK with the change.

Reply:
hum 2.16 is using 3.17.x, so I though we agree on bumping 2.17 to 3.18, then 2.18 or so to 3.19 ?

Reply:
My bad, I'm biased since working on master or very old branches. Indeed this would need to be done after we tag 2.17.

Reply:
I changed it to update version of kubernetes to 1.21.2 and helm to 3.6.1, that's all